public/content/staking/pools/index.md

```diff
@@ -53,7 +53,7 @@ There are a variety of options available to help you with your setup. Use the ab
 
 <StakingProductsCardGrid category="pools" />
 
-Please note the importance of choosing a service that takes [client diversity](/developers/docs/nodes-and-clients/client-diversity/) seriously, as it improves the security of the network, and limits your risk. Services that have evidence of limiting majority client use are indicated with <em style="text-transform: uppercase;">"execution client diversity"</em> and <em style="text-transform: uppercase;">"consensus client diversity."</em>
+Please note the importance of choosing a service that takes [client diversity](/developers/docs/nodes-and-clients/client-diversity/) seriously, as it improves the security of the network, and limits your risk. Services that have evidence of limiting majority client use are indicated with <em style={{ textTransform: "uppercase" }}>"execution client diversity"</em> and <em style={{ textTransform: "uppercase" }}>"consensus client diversity."</em>
```

Comment:
@pettinarip Encountered the same thing here as I did on the roadmap pages where I got a compiler error for these files, when attempting to pass a string to `style` instead of an object.